pkg/ccl/backupccl/show.go

```diff
@@ -227,6 +228,21 @@ func showBackupPlanHook(
 		}
 		defer store.Close()
 
+		isRoot, err := IsCollectionRoot(ctx, store)
```

Comment:
I think this check needs to be in `ReadBackupManifestFromStore` which is where the shower first attempts to read the backup manifest file. In the error condition of that method, if we get a file not exists error, then we check:

- the initial command was a `SHOW BACKUP`
- the LATEST file exists

If both are true we return an error but wrapped with a hint. Look at other places that use `errors.WithHintf`. The verbiage of the hint can be something along the lines of:

`is the path specified in `SHOW BACKUP` pointing to a collection? If so, use `SHOW BACKUPS IN` to locate specific backup subdirectories to be used with `SHOW BACKUP``

Reply:
actually, not in `ReadBackupManifestFromStore` since that is shared by actual backup execution as well. Lets move this logic to line 95 in `show.go`? So we intercept the error from `ReadBackupManifestFromStore` and perform the required checks.

Reply:
Done